docs/getting-started/zowe_faq.md

```diff
@@ -521,42 +521,32 @@ Install the plug-in in your IntelliJ-based IDE directly from marketplace or down
 
 </details>
 
-### Where can I use Zowe IntelliJ plug-in?
+### Where can I use Zowe Explorer plug-in for IntelliJ IDEA?
 
 <details className="zowe-faq">
 
 <summary></summary>
 
-You can use it in any IntelliJ-based IDE. 
+You can use it in any IntelliJ-based IDE, such as IntelliJ IDEA, PyCharm, Android Studio, etc.
 
 </details>
 
-### How do I get help with using Zowe IntelliJ plug-in?
+### How do I get help with using Zowe Explorer plug-in for IntelliJ IDEA?
 
 <details className="zowe-faq">
 
 <summary></summary>
 
-You can read detailed user guide and find any information you need [here](https://plugins.jetbrains.com/plugin/18688-zowe-explorer/user-guide). Also, you can ask any questions in the Zowe Slack channel [#zowe-explorer-intellij](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
+You can start with [use cases](../user-guide/intellij-use-cases.md) section to learn about use cases and to learn how to install and use the plug-in. Also, you can ask any questions in our [Zowe Slack channel (#zowe-explorer-intellij)](https://openmainframeproject.slack.com/archives/C020BGPSU0M).
```

Comment:
missing a word; capitalize the section title; remove one unnecessary "to learn": "You can start with **the** **U**se case section to learn about use case and how to install and ... "